app/src/main/res/values/strings.xml

```diff
@@ -90,6 +90,7 @@
     <string name="pref_exoplayer_start_landscape_video_in_landscape">Start landscape mode videos in landscape orientation</string>
     <string name="pref_exoplayer_allow_brightness_volume_gesture">Brightness and volume gestures</string>
     <string name="pref_exoplayer_remember_brightness">Remember display brightness</string>
+    <string name="pref_exoplayer_swipe_exclusion_size_vertical">Gesture dead zone</string>
```

Comment:
Not sure what the desired label would be here

Reply:
Seems fine as is.